OpenEXR/IlmImf/ImfScanLineInputFile.cpp

```diff
@@ -524,14 +524,14 @@ LineBufferTask::execute ()
     
         if (_lineBuffer->uncompressedData == 0)
         {
-            int uncompressedSize = 0;
+            size_t uncompressedSize = 0;
```

Comment:
Since size_t can be 32 bits, I think an int64_t or uint64_t here is appropriate. It's used below in the _lineBuffer->dataSize comparison, ->dataSize is a signed int, uncompressedSize should be a signed type as well, or dataSize should be updated.